.github/workflows/static.yml

```diff
@@ -81,14 +81,14 @@ jobs:
   #       run: errcheck
   #       comment: false
 
-  # sec:
-  #   name: Sec
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #   - uses: actions/checkout@master
-  #   - name: check
-  #     uses: grandcolline/golang-github-actions@v1.1.0
-  #     with:
-  #       run: sec
-  #       flags: "-exclude=G104"
-  #       comment: false
+  sec:
+    name: Sec
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: check
+      uses: grandcolline/golang-github-actions@v1.1.0
+      with:
+        run: sec
+        flags: "-exclude=G104,G101,G304"
```

Comment:
Comment on why we exclude these?

Reply:
because we have a lot of hardcoded in our test code have to exclude these warnings